zaza/openstack/charm_tests/mysql/tests.py

```diff
@@ -1145,3 +1145,75 @@ def test_910_restart_on_config_change(self):
             {}, {},
             self.services)
         logging.info("Passed restart on changed test.")
+
+    def test_action_bootstrap_mysqlrouter(self):
```

Comment:
I just noticed that this test name needs a number to fit in with the other numbered tests.  I suggest `test_920_bootstrap_mysqlrouter_action()` to fit in with the other numbered tests.

Reply:
Thanks I changed it